sdk/sqlvirtualmachine/Microsoft.Azure.Management.SqlVirtualMachine/src/Microsoft.Azure.Management.SqlVirtualMachine.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.SqlVirtualMachine</PackageId>
+		<Description>Microsoft Azure SqlVirtualMachine Library</Description>
+		<AssemblyName>Microsoft.Azure.Management.SqlVirtualMachine</AssemblyName>
+		<Version>1.1.0</Version>
+		<PackageTags>Microsoft Azure Library;SqlVirtualMachine;</PackageTags>
```

Comment:
Could you also provide a `<PackageReleaseNotes>`, because that's going to be displayed on Nuget Gallery.
[example](https://github.com/Azure/azure-sdk-for-net/blob/a01ba31940779d2be14c628c993dd8bd5b443946/sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj)

Reply:
I added the .txt file and PackageReleaseNotes. Thanks.